py-rse/ci.Rmd

```diff
@@ -108,168 +94,146 @@ knitr::include_graphics("figures/rse-ci/add-repo.png")
 knitr::include_graphics("figures/rse-ci/list-repos.png")
 ```
 
-### Configuring a project
+### Configuring a project for Travis
+
+In order for Travis to know what to do with the project repository,
+a file called `.travis.yml` must be present inside the repository.
+So, let's create this file!
 
-The next step is to create a file called `.travis.yml`
-in the root directory of the repository.
+Make the `.travis.yml` file in the root directory of the repository.
 (The leading `.` in the name hides the file from casual listings on Mac or Linux,
 but not on Windows.)
 This file contains settings that control Travis CI's operation,
 and is written in a format called [YAML][yaml] (Appendix \@ref(yaml)).
-A simple template `.travis.yml` configuration file looks like this:
+
+Now that we have a `.travis.yml` file, open it and add the following line:
+
+```yaml
+language: python
+```
+
+The `language` setting (or key) tells Travis CI which programming language to use
+so that it knows which of its standard [virtual machines][virtual-machine] to use
+as a starting point for the project.
```

Comment:
footnote pointing people at where they can find out more about what standard VMs are available and what's in them?